examples/tfengine/generated/org_foundation/audit/main.tf

```diff
@@ -71,9 +71,9 @@ module "bigquery_export" {
   source  = "terraform-google-modules/log-export/google"
   version = "~> 6.0.0"
 
-  log_sink_name          = "bigquery-audit-logs-sink"
+  log_sink_name          = var.logs_bigquery_dataset.sink_name
   destination_uri        = module.bigquery_destination.destination_uri
-  filter                 = "logName:\"logs/cloudaudit.googleapis.com\""
+  filter                 = "logName:\"logs/cloudaudit.googleapis.com\"${var.additional_filters}"
```

Comment:
I would suggest making `additional_filters` a `list(string)` and using the [join](https://www.terraform.io/docs/language/functions/join.html) function to concatenate them with the correct separator. Otherwise someone could add the filter incorrectly or without a space in the beginning for example.

Reply:
Wonderful. Thanks!